models/public/netvlad/netvlad.md

```diff
@@ -0,0 +1,94 @@
+# netvlad
+
+## Use Case and High-Level Description
+
+NetVLAD is a CNN architecture which tackles the problem of large scale visual place recognition. The architecture uses VGG 16 as base network and NetVLAD - a new trainable generalized VLAD (Vector of Locally Aggregated Descriptors) layer. It is a place recognition model pretrained on the [Pittsburgh](http://www.ok.ctrl.titech.ac.jp/~torii/project/repttile/) dataset.
+
+The model input is a blob that consists of a single image of "1x200x300x3" in RGB order.
+
+The model output is vector of "1x4096" descriptors which are used as image representation.
```

Comment:
not sure, do we need this info in description, anyway we have inputs/outputs section

Reply:
Removed